README.md

```diff
@@ -47,9 +47,9 @@ Files matching this pattern will be included in the manifest.
 
 _Default:_ `"**/*.{js,css,png,gif,ico,jpg,map,xml,txt,svg,swf,eot,ttf,woff,woff2}"`
 
-### ignoreFilePattern
+### fileIgnorePattern
 
-Files matching this pattern will _not_ be included in the manifest even if they match filePattern.
+Files matching this pattern will _not_ be included in the manifest even if they match filePattern. And thus will be allways deployed.
```

Comment:
@bortevik 
Yes, you are entirely correct. That is quite misleading and I'm not sure how that came to be the case. Those changes seem to have been introduced [in this commit](https://github.com/ember-cli-deploy/ember-cli-deploy-manifest/commit/fa5d82d305c214970f9db029231ba0bce5438525#diff-04c6e90faac2675aa89e2176d2eec7d8).

In my opinion, [the content that was there before](https://github.com/ember-cli-deploy/ember-cli-deploy-manifest/blob/79ea2c6bb3b2e9e286577a5e1a969964e5b8454a/README.md) made things a bit clearer.

Would you be interested in having a crack and seeing if you can improve the README at all?